app/code/Magento/Developer/Model/XmlCatalog/Format/PhpStorm.php

```diff
@@ -34,16 +31,13 @@ class PhpStorm implements FormatInterface
     private $domDocumentFactory;
 
     /**
-     * @param ReadFactory $readFactory
      * @param WriteFactory $fileWriteFactory
      * @param DomDocumentFactory $domDocumentFactory
      */
     public function __construct(
-        ReadFactory $readFactory,
```

Comment:
Unfortunately, we are not able to remove dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
check, missed out on that  one, did a crawl on code, didn't find any ;)